### PR TITLE
Add project tracking and conventions we want to adopt

### PR DESCRIPTION
**Pull request by moneyball**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -52,6 +54,8 @@ The Minimal Supported Rust Version is 1.22.0 (enforced by our Travis).
 Commits should cover both issues fixed and solutions' rationale.
 These [guidelines](https://chris.beams.io/posts/git-commit/) should be kept in mind.
 
+To facilitate communication with other contributors, if you are working on a GitHub Issue please assign it to yourself using GitHub's "assignee" field.
```
Comment (valentinewallace) [inline CONTRIBUTING.md]: nit: Issue -> issue

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ but also #rust-bitcoin on IRC Freenode.
 Discussion about code base improvements happens in GitHub issues and on pull
 requests.
 
+Major projects are tracked [here](https://github.com/rust-bitcoin/rust-lightning/projects). Major milestones are tracked [here](https://github.com/rust-bitcoin/rust-lightning/milestones?direction=asc&sort=title&state=open).
```
Comment (TheBlueMatt) [inline CONTRIBUTING.md]: Linebreaks at around 100 chars, please.

File: CONTRIBUTING.md (hunk 1 above)
Comment (TheBlueMatt) [inline CONTRIBUTING.md]: I don't actually know if non-"triage"-permissions folks can do this. Maybe just note that you should first check that no one is assigned and then comment suggesting that you're working on it, but don't hesitate to ask if the assigned party or previous commenters are still working on it if its been a while.

**Comment by TheBlueMatt** [comment]
Ping @moneyball

File: CONTRIBUTING.md (hunk 1 above)
Comment (moneyball) [reply CONTRIBUTING.md]: Addressed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (moneyball) [reply CONTRIBUTING.md]: Addressed.

File: CONTRIBUTING.md (hunk 1 above)
Comment (moneyball) [reply CONTRIBUTING.md]: Fixed

**Comment by moneyball** [comment]
I think it is ready for merge now.

**Pull request merged**